credentials/apps/credentials/apps.py

```
@@ -0,0 +1,40 @@
+"""
+Basic configuration for the credentials app.
+"""
+import logging
+
+from django.apps import AppConfig
+from django.conf import settings
+
+
+log = logging.getLogger(__name__)
+
+
+class CredentialsConfig(AppConfig):
+    """
+    Basic configuration validation.
+    """
+    name = 'credentials.apps.credentials'
+    verbose_name = 'Credentials'
+
+    def ready(self):
+
+        try:
+            settings.PROGRAMS_API_URL and settings.PROGRAMS_JWT_AUDIENCE and settings.PROGRAMS_JWT_SECRET_KEY
+        except AttributeError:
```

The point of this validation is to prevent the application from starting if any of these settings are missing. You are catching the exceptions, logging them, and continuing with program execution. Don't catch the exception. Let the app fail.